flake8_spellcheck/__init__.py

```diff
@@ -26,6 +26,8 @@ def detect_case(name):
         return "url"
     elif "_" in name:
         return "snake"
+    elif name.isupper():
+        return "snake"
```

Comment:
trying to think of any scenario where this wouldn't be snake case :thinking: but i think it should actually be ok